modules/nodes-cluster-worker-latency-profiles-using.adoc

```diff
@@ -7,9 +7,9 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="nodes-cluster-worker-latency-profiles-using_{context}"]
-= Using worker latency profiles
+= Using and changing worker latency profiles
 
-To implement a worker latency profile to deal with network latency, edit the `node.config` object to add the name of the profile. You can change the profile at any time as latency increases or decreases.
+To change a worker latency profile to deal with network latency, edit the `node.config` object to add the name of the profile. You can change the profile at any time as latency increases or decreases.
 
 You must move one worker latency profile at a time. For example, you cannot move directly from the `Default` profile to the `LowUpdateSlowReaction` worker latency profile. You must move from the `default` worker latency profile to the `MediumUpdateAverageReaction` profile first, then to `LowUpdateSlowReaction`. Similarly, when returning to the default profile, you must move from the low profile to the medium profile first, then to the default.
```

Comment:
Lets capitalize the second 'default' here to keep consistent

Reply:
@paigerube14 I capitalized and "literalized" both those later instances of "default" to `Default`